### PR TITLE
Logo component (fixes #10)

### LINKED ISSUE #10: Logo component

### PR DESCRIPTION
**Pull request by OtisRed**
It does everything that I would expect it to do and since displaying logo isn't really a rocket science I think it's ready :) 

I hard-coded max-height in px because without it I guess there was no reference point for .svg image in the fluid container and it was stretching down beyond the screen-size display.

### CODE REVIEW

**Comment by stanislawK** [comment]
Looks ok for me, but could you change pr and branch name follow the working agreement: https://bit.ly/2H7pAjs ?

File: frontend/src/components/HomePage.vue [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <template>
-  <div>
+   <div>
```
Comment (w1stler) [inline frontend/src/components/HomePage.vue]: Why 3 spaces, not 2?

File: frontend/src/components/Logo.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<template>
+  <div>
+    <v-app>
+      <v-container fluid id="logo-page">
+        <v-layout align-center justify-center fill-height>
+          <v-img
```
Comment (w1stler) [inline frontend/src/components/Logo.vue]: Why not in one line?

**Comment by OtisRed** [comment]
Actuallly we decided to lose this component altogether so I'm closing it

**Pull request closed without merging**